CHANGELOG.md

```diff
@@ -2,30 +2,35 @@
 
 All notable changes to `nova-enum-field` will be documented in this file.
 
-## 1.0.0 - 2019-09-07
+## 2.0.0 - 2020-07-08
```

Comment:
I reordered this to be newest-oldest. Saves having to scroll down to see the lastest release and in line with most other changelogs.

Reply:
This is great, thanks!